backend/de.metas.business/src/main/java/org/eevolution/api/impl/ProductBOMDAO.java

```diff
@@ -419,6 +419,20 @@ public Optional<I_PP_Product_BOM> getPreviousVersion(final @NonNull I_PP_Product
 				.firstOptional(I_PP_Product_BOM.class);
 	}
 
+	@Override
+	@NonNull
+	public List<I_PP_Product_BOM> retrieveAllBomVersions(final @NonNull I_PP_Product_BOM bomVersion)
```

Comment:
either u call this method sometihng like getSiblings
or you remove 
> .addNotEqualsFilter(I_PP_Product_BOM.COLUMNNAME_PP_Product_BOM_ID, bomVersion.getPP_Product_BOM_ID())

and u handle this in your caller.

----

Also
> final @NonNull I_PP_Product_BOM bomVersion

pls rename it to `bom` because it's NOT a `bomVersions`